src/containers/styles/Wise.scss

```diff
@@ -1,110 +1,130 @@
+/** @format */
```

Comment:
I think this annotation is causing a change in formatting. If you remove this annotation and run `npm run format`, I think the spacing changes will go back to normal.